src/components/Permissions/AppPermissions/AppPermissions.jsx

```diff
@@ -108,28 +108,49 @@ const AppPermissions = ({ t }) => {
         </Typography>
       ) : (
         <div>
-          <IconButton className="u-mr-half" href="#/permissions">
-            <Icon icon={PreviousIcon} size={16} />
-          </IconButton>
-          <div className="u-flex u-flex-column u-flex-items-center">
-            <div style={{ width: '48px' }}>
-              <AppIcon app={slugName} />
+          <div
+            className={
+              isMobile || isTablet
+                ? 'u-flex u-flex-column u-flex-items-center'
+                : 'u-flex u-flex-justify-between'
+            }
+          >
+            <div
+              className={
+                isMobile || isTablet ? '' : 'u-flex u-flex-items-center'
+              }
+            >
+              {!isMobile && !isTablet && (
+                <IconButton className="u-mr-half" href="#/permissions">
+                  <Icon icon={PreviousIcon} size={16} />
+                </IconButton>
+              )}
+              <div
+                className={isMobile || isTablet ? '' : ''}
```

Comment:
Both ternary parts return the same `''` value. Should we remove the `className` prop here?